dogFood.js

```diff
@@ -0,0 +1,27 @@
+var inputEl= document.getElementById("input");
+var getDogFood = new XMLHttpRequest();
+getDogFood.addEventListener("load", printToDomDog);
+getDogFood.open("GET", "dogFood.json");
+getDogFood.send();
+
+function printToDomDog () {
+	var dogFood= JSON.parse(this.responseText);
+	var string="";
+	// For loops to add to DOM
+	// Brand Names
+	for (i=0; i<dogFood.dog_brands.length; i++) {
+		string+= `<div class='col-md-6'><h2>${dogFood.dog_brands[i].name}</h2>`;
```

Comment:
A helpful practice when doing a bunch of nested loops is to save off the element you're at in each layer of the loop as a variable for later reference.  This has two benefits:
- It only makes a single grab of that data from the object instead of pulling from the object every call
- It drastically simplifies further deeper calls to the object
For example, here, if you saved off something like ```var brand = dogFood.dog_brands[i]``` you could refer to it in the HTML building string as just ```brand.name``` and likewise...